crest/Assets/Crest/Crest/Shaders/OceanInputs/AnimWavesScaleByFactor.shader

```diff
@@ -0,0 +1,123 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// This writes straight into the displacement texture and sets the water height to the y value of the geometry.
+
+Shader "Crest/Inputs/Animated Waves/Scale By Factor"
+{
+	Properties
+	{
+		[Enum(ColorWriteMask)] _ColorWriteMask("Color Write Mask", Int) = 15
```

Comment:
This may cause confusion but if you have a use case where only some channels should be scaled then I'm not against it

Reply:
I cannot. It's now removed.